pystemd/run.py

```diff
@@ -229,6 +234,10 @@ def bus_factory():
                 or None,
             }
         )
+        if user is DynamicUser:
+            unit_properties["DynamicUser"] = "yes"
```

Comment:
this has to be `unit_properties["DynamicUser"] = True` because DynamicUser is a boolean